tensorflow/python/ops/numpy_ops/np_array_ops.py

```diff
@@ -28,6 +28,7 @@
 from tensorflow.python.framework import dtypes
 from tensorflow.python.framework import ops
 from tensorflow.python.framework import tensor_shape
+from tensorflow.python.framework import tensor_util
```

Comment:
Hey @DarrenZhang01 I think this needs to be removed since its no longer used.

Reply:
Thanks for pointing this out, Akshay!